compose/frontend.yml

```diff
@@ -18,6 +18,51 @@ services:
     networks:
       - ccd-network
 
+  ccd-admin-web:
+    image: "${CCD_ADMIN_WEB_USE_LOCAL-hmctspublic.azurecr.io/}ccd/admin-web:${CCD_ADMIN_WEB_TAG:-latest}"
+    container_name: ccd-admin-web
+    environment:
+      SECURE_AUTH_COOKIE_ENABLED : "false"
+      USE_CSRF_PROTECTION : "false"
+      SECURITY_REFERRER_POLICY : origin
+      IDAM_BASE_URL: "${IDAM_STUB_SERVICE_NAME:-http://idam-api:5000}"
+      IDAM_S2S_URL: http://service-auth-provider-api:8080
+      IDAM_ADMIN_WEB_SERVICE_KEY: AAAAAAAAAAAAAAAD
+      IDAM_LOGOUT_URL: "${IDAM_STUB_LOCALHOST:-http://localhost:3501}/login"
+      IDAM_OAUTH2_TOKEN_ENDPOINT: "${IDAM_STUB_SERVICE_NAME:-http://idam-api:5000}/oauth2/token"
+      IDAM_OAUTH2_LOGOUT_ENDPOINT: "${IDAM_STUB_SERVICE_NAME:-http://idam-api:5000}/session/:token"
+      IDAM_OAUTH2_AW_CLIENT_SECRET: IIIIIIIIIIIIIIII
+      IDAM_SERVICE_NAME: ccd_admin
+      IDAM_OAUTH2_CLIENT_ID : ccd_admin
+      ADMINWEB_LOGIN_URL: "${IDAM_STUB_LOCALHOST:-http://localhost:3501}/login"
+      ADMINWEB_IMPORT_URL: http://ccd-definition-store-api:4451/import
+      ADMINWEB_JURISDICTIONS_URL : http://ccd-definition-store-api:4451/api/data/jurisdictions
+      ADMINWEB_USER_PROFILE_URL : http://ccd-user-profile-api:4453/users
+      ADMINWEB_SAVE_USER_PROFILE_URL : http://ccd-user-profile-api:4453/users/save
+      ADMINWEB_USER_ROLE_URL : http://ccd-definition-store-api:4451/api/user-role
+      ADMIN_ALL_USER_ROLES_URL : http://ccd-definition-store-api:4451/api/user-roles
+      ADMINWEB_CREATE_DEFINITION_URL : http://ccd-definition-store-api:4451/api/draft
+      ADMINWEB_UPDATE_DEFINITION_URL : http://ccd-definition-store-api:4451/api/draft/save
+      ADMINWEB_DELETE_DEFINITION_URL : http://ccd-definition-store-api:4451/api/draft
+      ADMINWEB_DEFINITIONS_URL : http://ccd-definition-store-api:4451/api/drafts
+      ADMINWEB_WHOAMI_URL : http://ccd-definition-store-api:4451/api/idam/profile
+      ADMINWEB_AUTHORIZATION_URL : http://ccd-definition-store-api:4451/api/idam/adminweb/authorization
+      ADMINWEB_IMPORT_AUDITS_URL : http://ccd-definition-store-api:4451/api/import-audits
+      ADMINWEB_ROLES_WHITELIST : "ccd-import,ccd-import-validate"
+      APPINSIGHTS_INSTRUMENTATIONKEY: key
+    ports:
+    - 3100:3100
+    depends_on:
+    - service-auth-provider-api
+    - ccd-definition-store-api
+    - ccd-user-profile-api
+    links:
```

Comment:
linking is an obsolete feature of docker. Better not using it. Can you delete this whole link section please? I think that won't break the Admin Web because communication with the other services happens now through the `ccd-network`